packages/block-library/src/template-part/edit/placeholder.js

```diff
@@ -28,8 +28,8 @@ function TemplatePartPreview() {
 }
 
 export default function TemplatePartPlaceholder( { setAttributes } ) {
-	const [ slug, _setSlug ] = useState();
-	const [ theme, setTheme ] = useState();
+	const [ slug, _setSlug ] = useState( '' );
+	const [ theme, setTheme ] = useState( '' );
```

Comment:
It was necessary to edit these to stop a react warning from breaking the e2e tests.  Without initializing as an empty string, react would complain about turning an uncontrolled component into a controlled component once the input was used.